fluentbit-kafka-fluentd/README.md

```diff
@@ -0,0 +1,173 @@
+# Face to Face 2020/03 - Fluentbit -> Kafka -> Fluentd -> Elasticsearch
+
+## What?
+The purpose of this hackathon item was to investigate a new collector/aggregator solution for Cluster Logging.
+
+## Why?
+Currently Fluentd takes up decent amount of CPU and RAM <sup>[1](#comparison)</sup> on the host nodes in order to collect, enrich, and ship logs.
+
+Fluentbit is a slimmer version of Fluentd (created by the same company) and is written in C. It does not have the same amount of plugins as Fluentd, however we felt it was best to just let it pull logs from the nodes and ship them to the aggregator as fast as possible.
+
+Using Kafka lets us add reliability into our log delivery stack. Each of these components can be configured such that there are `ack`s for as messages are sent to the next component, ensuring that our messages are being persisted before we know we no longer require them. Leveraging Kafka also adds as an additional buffer for our logs to be stored in, easing the potential burden on our aggregator without as much worry for loss of logs.
```

Comment:
@syedriko thoughts?

Reply:
+1. We could add that Kafka decouples the final destinations of logs from the log collectors allowing for possible downtime of those destinations due to maintenance, lack of connectivity, etc. At the moment this capability only exists in the form of buffering in the log collectors, which is limited.

Reply:
@syedriko how does this new paragraph sound?

Reply:
Sounds great!